src/script/backport-create-issue

```diff
@@ -159,11 +159,16 @@ def update_relations(r, issue, dry_run):
                 "tracker")
             continue
         if relation['relation_type'] != 'copied_to':
-            logging.error(self.url(issue) + " unexpected relation '" +
-                relation['relation_type'] + "' to " + self.url(other))
+            logging.error(url(issue) + " unexpected relation '" +
+                relation['relation_type'] + "' to " + url(other))
             success = False
             continue
         release = get_release(other)
+        if release in existing_backports:
+            logging.error(url(issue) + " duplicate " + release +
```

Comment:
yeah, we should move on in this case. i am just wondering if we really need to return a "success" here?